### ISSUE TITLE
Branding guidelines?

### ISSUE THREAD

**Issue by acli** [citation-file-format/branding#2]
Are there any branding guidelines? Should we create one?

Are there any restrictions on who can use the logo / how the logo should be used? Are they under some kind of CC license?

**Comment by sdruskat** [comment]
Thanks, @acli, for this issue!

To be honest, the current "branding" was a quick shot from the hip at a logo to disambiguate the GitHub organization repositories in the repo overview for us internally, mostly.

I'd quite like to have a nice logo and branding that we can use here, on the website, and in slides, etc., but we're no graphic designers. I think to have branding guidelines, we should have a logo and a - what do you call it? - wordmark.

FWIW, I like the [CSL logo](https://citationstyles.org/assets/img/cropped-csl-logo-600-bordered.png) and the [de-RSE logo](https://raw.githubusercontent.com/DE-RSE/logo-association/master/concept/de-RSE-logo-text-colour.png).

@jspaaks, as the creator, do you have any thoughts on licensing the current logo?

**Comment by acli** [comment]
Nah, @sdruskat, you don’t absolutely *need* a wordmark to have branding guidelines (in any case, I’d say the current logo is already technically a wordmark). I didn’t graduate from graphic design but I’d be pretty confident to say you just need a desire to lay down things like what it means to be (visually) consistent, how consistent you want things to be, etc. We could start with just a simple list (I know one-page branding guidelines exist out there)

**Comment by sdruskat** [comment]
Thanks @acli, I think this sounds reasonable. Does it still make sense to create a new logo before drafting guidelines?

**Comment by acli** [comment]
@sdruskat I’m probably not the right person to answer that question (I usually don’t touch branding projects – it’s not my area), but as far as I understand this partly depends on whether the current logo already has “equity” (whether it’s well-known).

**Comment by sdruskat** [comment]
> this partly depends on whether the current logo already has “equity” (whether it’s well-known).

I daresay it's likely either not well-known, or completely unknown :slightly_smiling_face:.
I'd be happy to leave this issue open until we have a new logo (although I can't say when this is going to be, depends on workload or finding volunteers)?

**Comment by jspaaks** [comment]
I think it is probably helpful to write down a couple of pointers on how we would like people to (not) use the CFF branding. Generally speaking the wording should give the impression of not being too restrictive. Maybe the GitHub octocat logo text could be an example? https://github.com/logos

For now, I'll stick a ~~CC-BY-4.0~~CC0 on this repo, so people can at least legally clone the repo if they feel like it :)

**Comment by acli** [comment]
I have forked the repo and created [a branch to work on this](https://github.com/acli/branding/tree/20210829_DRAFT_GUIDELINES_WORKING). Please comment on [the problem statement](https://github.com/acli/branding/blob/20210829_DRAFT_GUIDELINES_WORKING/doc/PROBLEM_STATEMENT.md) and [the draft guidelines](https://github.com/acli/branding/blob/20210829_DRAFT_GUIDELINES_WORKING/README.md). Thanks.

I’m going to create a separate branch to work on small tweaks to the logo (specifically the specific tint of blue, legibility checks, and visual centring, i.e. points 1, 2, 4 in the problem statement).

**Comment by jspaaks** [comment]
Hi Ambrose, 
Just a couple of notes:
- I changed the font color to #FFFFFF in the last iteration, seemed easier
- The fonts were converted to paths to avoid problems with showing the svg logo in some places that didnt have the Roboto Condensed font
- We are still planning to update the logo, potentially to something rather different, so be mindful of how much time/effort you want to invest 

Regarding the guidelines and problems statement:
- Nice additions here https://github.com/acli/branding/blob/20210829_DRAFT_GUIDELINES_WORKING/README.md, thanks!
- Legibility for slides was not tested
- Re bullet point 4 from https://github.com/acli/branding/blob/20210829_DRAFT_GUIDELINES_WORKING/doc/PROBLEM_STATEMENT.md: if you mean how I included the image [here](https://github.com/citation-file-format/branding/commit/c98ad41cc854c8db52fdaa2dd237463cc1f28456), that's expected. Centering could be added with HTML, for example like Ive done here https://github.com/zenodraft/zenodraft/blob/0.11.0/README.md

Perhaps you can already make a Pull Request while you work on things? That'll make it easier to see the differences. Just mark it Draft so we'll know it's still a work in progress.

**Comment by acli** [comment]
Hi @jspaaks ,

Re bullet point 4, I mean centring the letters “CFF” within the rounded square. Currently it’s probably mechanically centred but I don’t feel it’s visually centred, and in logo design we would care more about visual centring. “CFF” is a little hard to visually centre because the curves on the left of the C and the whitespace on the right side of the F pull our attention in opposite directions, but in my judgement “CFF” should be moved slightly to the left. I’ll show you in a pull request once I have it ready. (The pull request will be just for demonstration; I don’t have the correct software.)

Let’s treat this as an exercise in requirements analysis.

**Comment by acli** [comment]
I have created a demo at https://github.com/acli/branding/blob/20210830_LOGO_TWEAK_DEMO/doc/TWEAKS.md

**Comment by jspaaks** [comment]
> care more about visual centring

agree

Re software, I used Inkscape https://inkscape.org/

I'll try to look at the demo later

**Comment by sdruskat** [comment]
Latest thoughts from the discussion about governance:

- Only (sub-)projects in the CFF project should be allowed to use our (future) branding, and we encourage them to do it, but won't force them to.
